velox/functions/prestosql/Rand.h

```diff
@@ -28,6 +28,33 @@ struct RandFunction {
     result = folly::Random::randDouble01();
     return true;
   }
+
+  FOLLY_ALWAYS_INLINE void call(int64_t& out, const int64_t& input) {
```

Comment:
It might be less code to turn this into a template:

```
template <typename N>
void call(int64_t& out, N n)
```

Also, pass integers by value, not by reference.